.coderabbit.yml

```
@@ -0,0 +1,43 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+# https://docs.coderabbit.ai/guides/configure-coderabbit/
+
+language: "en-US"
+early_access: true
+reviews:
+  profile: "chill"
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: false
+  review_status: false
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: false
+    auto_incremental_review: false
+    base_branches:
+      - master
+      - beta
+      - rc
+  path_instructions:
+    - path: "**/*"
+      instructions: "
+      Focus on code smells, logic errors, edge cases, missing test cases, security flaws and serious issues.
+      Avoid commenting on minor issues such as linting, formatting and style issues.

```

Some suggestions, particularly regarding handling of documentation.
```suggestion
      Avoid commenting on minor issues such as linting, formatting and style issues.
      Check spelling of changes in markdown (*.md)files and suggest improvements in spelling/grammar where appropriate.
      Ensure that changes in markdown files (*.md) follow the style of one sentence per line, suggest changes if this is not met.
```

---

There's a default setting currently enabled for spell checking, with much more complex configuration available, but the sentence per line mention is worth adding

---

I've fixed this up now